mobileversion.html

```diff
@@ -0,0 +1,248 @@
+               <ul class="mainp">
+                  <li class="canopy">CANOPY</li>
+                  <li class="mid">&#8226;</li>
+                  <li class="back">Back End Dev</li>
+                  <li class="mid">&#8226;</li>
+                  <li class="year">2015</li>
+                  </ul>
+            </div>
+            <div class="day">
+               <p class="daily">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+               </p>
+            </div>
+            <div class="maincon">
+               <ul>
+                  <li class="container2"><a href="#">html</a></li>
+                  <li class="container3"><a href="#">css</a></li>
+                  <li class="container4"><a href="#">javaScript</a></li>
+               </ul>
+            </div>
+            <div class="sub"><button class="sbmt" type="submit">See Project </button></div>
+         </div>
+      </div>
+      </div>
+   </section>
+   <section>
+      <div id="About" class="About">
+         <h2 class="Abouth2">About<br>
+            Myself</h2>
+         <div>
+            <p class="Hello">Hello I’m a software developer! I can help you build a product , feature or website Look
+               through some of my work and experience! If you like what you see and have a project you need coded,
+               don’t hestiate to contact me.</p>
+         </div>
+         <div>
+            <h3 class="Lets">Let's Connect</h3>
+         </div>
+         <div >
+            <ul class="fonticons">
+               <li><a href="#"><i class='fab fa-github'></i></a></li>
+               <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+               <li><a href="#"><i class='fab fa-angellist'></i></a></li>
+               <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+               <li><a href="#"><i class='fab fa-medium-m'></i></a></li>
+            </ul>
+         </div>
+         <div class="get">
+            <a href="#">Get my resume</a>
+         </div>
+         <div>
+            <h2 class="lang">Languages <i class="fas fa-caret-down"></i></h2>
+         </div>
+         <div class="grid">
+            <ul>
+               <li class="griditem1">
+                  <h3 class="Script"><img
+                        src="https://ih1.redbubble.net/image.815350031.4911/flat,128x128,075,t.u1.jpg">JavaScript
+                  </h3>
+               </li>
+               <li class="griditem2">
+                  <h3 class="HTML"><img
+                        src="https://cdn-icons-png.flaticon.com/128/919/919827.png?ga=GA1.2.799972565.1666983033">HTML
+                  </h3>
+               </li>
+               <li class="griditem3">
+                  <h3 class="CSS"><img
+                        src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/3d/CSS.3.svg/428px-CSS.3.svg.png?20160504153216CSS">CSS
+                  </h3>
+               </li>

```

Comment:
- [ ] [Optional] I would suggest you, please stick to the design as much as possible. Please fix every spacing. See the image below for reference
### Your design
![his](https://user-images.githubusercontent.com/81727988/205065324-c5788224-3123-4a27-b6e6-6d1b2b97cdc9.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/205065368-19e7c7e0-20f7-4bd6-9527-6d1048de17e9.png)